packages/cli/src/index.ts

```diff
@@ -522,7 +522,9 @@ const main = async () => {
         return 1;
       }
 
-      output.error('Not able to load user');
+      output.error(
+        `Not able to load user because of unexpected error code: ${err.code}`
+      );
```

Comment:
Probably need something like this:

```ts
      if (isErrnoException(err))
        output.error(
        `Not able to load user because of unexpected error code: ${err.code}`
        );
      }
      
     output.error(
        `Not able to load user because of unexpected error: ${err}`
      );
```